[data sanitization service] component deployment

matthewrossi opened the pull request:
This PR, related to Issue https://github.com/glaciation-heu/IceStream/issues/119, includes the deployment of three services:
- Spark operator: Kubernetes operator for managing the lifecycle of Apache Spark applications on Kubernetes
- Data sanitization service: it creates Spark applications on demand when the user needs to sanitize datasets
- Spark history server: it offers a Web UI providing insights about the data sanitization processes run by the data sanitization service

a-zharinov reviewed (changes requested):
Matthew, thank you for the pull-request. You did a good job.

I see the only major issue. We have an agreement to use Helm chart as deployment units. Please envelop Data Sanitization and Spark History services in Helm charts. I recommend you to do it original repositories of these components.

matthewrossi commented:
With the new commits I have updated the deployment of the data sanitization service and Spark history server to use Helm charts and added the new services to the integration list. Please let me know if I can proceed merging the PR.

matthewrossi commented on integration/apps/security/data-sanitization-service.yaml:
```diff
@@ -0,0 +1,182 @@
+  ingressClassName: nginx
+  tls:
+  - hosts:
+    - data-sanitization.integration
```
I should have fixed this with commit 93ece26

matthewrossi commented:
@a-zharinov any chance of bringing this forward?

I have addressed the requested changes, but I need a final check to proceed with the merge.
